public/individualdevstats.php

```diff
@@ -412,6 +412,15 @@
 $maxRecentAchievements = 200;
 $recentlyObtainedAchievements = getRecentUnlocks($achievementIDs, 0, $maxRecentAchievements);
 
+// Initialize leaderboard variables
+$userLeaderboardCount = 0;
+
+// Get leaderboard information for user
+$leaderboards = getLeaderboardCounts($dev);
+foreach ($leaderboards as $game) {
+    $userLeaderboardCount += $game['LeaderboardCount'];
+}
```

Comment:
With #1474, you should be able to tally these up from $gamesList (~line 83) to avoid adding the new function and extra query.

Reply:
This sounds like a good improvement - @cIymax would you be willing to make this change?
Then we could merge this PR.

Reply:
The reason why I added the new function even with #1474 is so that we pull games for which the dev made zero achievements but some leaderboards for.  My understanding of #1474 is that it still wouldn't pull such games, hence the added function. @wescopeland, as the author of #1474, can you confirm my understanding is correct?

Reply:
To the best of my recollection, that understanding is correct.

https://github.com/RetroAchievements/RAWeb/blob/cf5040e0b4206d27f8017f7fa235364f15ffb3ba/app/Helpers/database/game.php#L294C5-L294C5

Reply:
I see.  Let me know if/how I can assist.  I see a "Resolve conflicts" button below, but it's grayed out for me.